RELEASE.md

```diff
@@ -133,18 +133,17 @@ This repo includes a "development" chart in the [charts/](./charts/fluent-operat
 
 Fluent Operator uses [custom builds](./cmd/fluent-watcher/README.md) of both Fluentd and Fluent-bit.  These images can (and often should be) be published out-of-band of Fluent Operator releases.
 
-## Fluent-bit 
+## Fluent-bit
 
 To publish a new fluent-bit image:
 
-* Execute the [bump-fluent-bit-version](https://github.com/fluent/fluent-operator/actions/workflows/bump-fluent-bit-version.yaml) workflow dispatch to generate a PR to update fluent-bit version references in this repo 
-* Merge the PR generated by the [bump-fluent-bit-version](https://github.com/fluent/fluent-operator/actions/workflows/bump-fluent-bit-version.yaml) workflow
-* Execute the [build-fb-image](https://github.com/fluent/fluent-operator/actions/workflows/build-fb-image.yaml) workflow dispatch to build and publish the new image 
+* Execute the [bump-fluent-bit-version](https://github.com/fluent/fluent-operator/actions/workflows/bump-fluent-bit-version.yaml) workflow dispatch to generate a PR to update fluent-bit version references in this repo
+* Merge the PR generated by the [bump-fluent-bit-version](https://github.com/fluent/fluent-operator/actions/workflows/bump-fluent-bit-version.yaml) workflow. (This will [automatically trigger the release](https://github.com/fluent/fluent-operator/blob/master/.github/workflows/build-fb-image.yaml#L4-L8) of the new fluent-bit image)
```

Comment:
This last hyperlink will point to the exact line that would trigger the workflow automatically after merge.

Also see https://github.com/fluent/fluent-operator/pull/1515/files#diff-09a495bd7e847d10598ae03091d49e7fe7ed42ab64ae6431594f123a3419329bR4-R8